### ISSUE TITLE
Element appearance changes after Archi restart

### ISSUE THREAD

**Issue by srivaths** [archimatetool/archi#246]
Elements such as **Application Component**, **Node**, **Device** etc have two different representations.  For example, **Application Component** can be represented as 
![archi](https://user-images.githubusercontent.com/27290/27004017-3e0752b6-4dd0-11e7-9d5a-16c343b78cce.png)
or as 
![archi_1](https://user-images.githubusercontent.com/27290/27004043-6269311a-4dd0-11e7-9b4b-9cfdb3c08209.png)

I happen to prefer the first option.  However upon restarting Archi, all **Application Component** instancs revert to the second style.  This issue affects **all** diagram elements that have multiple representation styles.

This happen _even though_ 

(a) I have stated my preference (Preferences > Diagram > Default Figures)
(b) My preference happens to match Archi-default

It appears that Archi is not only ignoring _my preference_, it is even ignoring the its own defaults.

In the grand scheme of things this could be considered a minor issue except for the fact that the change in appearance messes up the layout of the diagram.  Also, reverting to _my preference_ is not an easy matter of selecting all **Application Component** instances or all **Device** instances and changing the figure.  You **have to do it** one at a time.

**Comment by Phillipus** [comment]
Version of Archi?

**Comment by srivaths** [comment]
So sorry for leaving out that key piece of information.

I am using Archi 4.0.0 (Build  201705031219) on macOS (10.12.5)

**Comment by Phillipus** [comment]
At the moment I can't reproduce this. Please create a new simple model using Archi 4.0.1 with one element in and try again.

**Comment by srivaths** [comment]
Before I installed Archi 4.0.1 I thought I would try your experiment with a new model in the version I already have installed.

After several experiments I was left with inconsistent results.  Sometimes the element representation changed and sometimes not.  It seemed to make a difference if I dragged an element from the pallet vs. made a copy of an existing element.

Then I noticed that the default setting for some elements had changed.  I reset them to match my preference.

Now I cannot reproduce the problem. :-(

Feel free to close this issue.
